.docksal/etc/conf/settings.kanopi.php

```diff
@@ -1,15 +1,26 @@
 <?php
 
+// phpcs:ignoreFile
+
 /**
  * @file
  * Local settings that are copied via fin init.
  */
 
-// Docksal DB connection settings.
+// Docksal Default DB connection settings.
 $databases['default']['default'] = [
   'database' => 'default',
-  'username' => 'user',
-  'password' => 'user',
+  'username' => 'root',
+  'password' => 'root',
+  'host' => 'db',
+  'driver' => 'mysql',
+];
+
+// Docksal Migration DB connection settings.
+$databases['migrate']['default'] = [
```

Comment:
I think we can remove this as the connection is coming from kanopi/drupal-integrations

Reply:
We should not remvoe this entry.
This is in place because on local environments we do not want to use the removte database connection that is set up in `settings.migration.php` in the `drupal-integration` module. 
This overrides that one to use the local database we set up using our custom `migrate-prep-db` command.